.golangci.yml

```diff
@@ -1,29 +1,40 @@
----
-linters-settings:
-  govet:
-    disable:
-      - shadow  # default value recommended by golangci
-      - composites
-  gosec:
-    excludes:
-      - G101
-
-linters:
-  enable:
-    - gosec
-
+version: "2"
 run:
   build-tags:
     - integration
-  timeout: 5m
-
+linters:
+  enable:
+    - gosec
+  settings:
+    gosec:
+      excludes:
+        - G101
+    govet:
+      disable:
+        - shadow
+        - composites
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      - linters:
+          - gosec
+        path: ^(images/)?(pkg|internal)/.*_test\.go$
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
 issues:
-  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
   max-issues-per-linter: 0
-
-  # Maximum count of issues with the same text. Set to 0 to disable. Default
-  # is 3.
   max-same-issues: 0
-  exclude-rules:
-    - path: ^(images/)?(pkg|internal)/.*_test\.go$
-      linters: ["gosec"]
+formatters:
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
I used `golangci-lint migrate` for this, can put the two comments back if you think it holds any value, I am not fan of repeating what is in documentation. It was also inconsistent - most of other config values had no comments anyway.

Reply:
I don't think we need to keep those comments.

Where did these exclusion paths come from though?  Probably built-in defaults?  Let's drop them and add any we might need later.  Right now, we want all `.go` files to be considered.